modules/machineset-delete-policy.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * machine_management/manually-scaling-machineset.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="machineset-delete-policy_{context}"]
+= The MachineSet delete policy
```

Comment:
You could change "MachineSet" to "machine set" in line with the new API object reference guidelines.

Reply:
I am also wondering whether "deletion policy" or "delete policy" sounds more fluent. Unless you are referring to the specification parameter in back ticks specifically (i.e. `deletePolicy`), you could consider using "deletion policy". There are a number of other instances of "delete policy" throughout the PR.